msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -95,7 +95,33 @@ public PublicClientApplication(string clientId, string authority)
             {
                 ClientId = clientId
             };
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="scopes"></param>
+        /// <param name="extraQueryParameters"></param>
+        /// <param name="deviceCodeResultCallback"></param>
+        /// <returns></returns>
+        public async Task<AuthenticationResult> AcquireTokenAsync(

```

Comment:
>AcquireTokenAsync [](start = 48, length = 17)

not sure this flow should be made available on mobile (as those have browsers)... though not sure I can see a big issue in having it there...